db/migrations/00025_create_storage_diffs_table.sql

```diff
@@ -0,0 +1,14 @@
+-- +goose Up
+CREATE TABLE public.storage_diff
+(
+    id             SERIAL PRIMARY KEY,
+    block_height   BIGINT,
+    block_hash     BYTEA,
+    hashed_address BYTEA,
+    storage_key    BYTEA,
+    storage_value  BYTEA,
+    UNIQUE (block_height, block_hash, hashed_address, storage_key, storage_value)
+);
+
+-- +goose Down
+DROP TABLE public.storage_diff;
```

Comment:
↵

Reply:
Are you suggesting an extra line after this one? I ran `Reformat Code` already but not sure if I'm missing something